code/TaxonomyTerm.php

```diff
@@ -32,6 +33,11 @@ class TaxonomyTerm extends DataObject implements PermissionProvider
 
     private static $default_sort = 'Sort';
 
+    private static $summary_fields = array(
+        'Name' => 'Name',
+        'getTaxonomyType' => 'Type'
```

Comment:
For getters just use the property name `TaxonomyType` not the method of the getter.